### PR TITLE
Fixing the failing build

### PR DESCRIPTION
**Pull request by pyepye**
I went to start using this package today but ran into a few issues straight away...

1.  `.ImproperlyConfigured: Specifying a namespace in include()` which appears to be fixed (https://github.com/dj-paddle/dj-paddle/pull/18) but is not on pip yet
2. When running `djpaddle_sync_plans_from_paddle` I was getting `malformed API response. "response" missing.` without any more information. It turns out my Vendor ID was copied badly but I was not getting the correct error - Again it appears to be fixed (https://github.com/dj-paddle/dj-paddle/pull/12) but not on pip yet

In other words I was hoping to get a new version on pip so I don't have to monkey patch the above.

I then noticed that the builds were failing on Travis so I decided to fix it all up as I thought that could be a blocker?

There are quite a few changes.
* A lot of it is import and linting fixes automatically done by `black` and `isort`
* I've tried to change as little code as possible. I did move some bits in utils to settings as I felt it would be better for it to fail on load
* Improved test coverage. Please check the tests are OK as it's been a long time since I wrote unittest style tests not straight pytest
* I'm not a big fan of having txt files instead of JSON files for fixtures but after checking out https://vendors.paddle.com/webhook-alert-test I thought it was best that the test data was as accurate to Paddles webhooks as possible

Please take a look and let me know what you think

Tox output:
```
---------- coverage: platform darwin, python 3.7.5-final-0 -----------
Name                                                              Stmts   Miss Branch BrPart  Cover
---------------------------------------------------------------------------------------------------
djpaddle/__init__.py                                                  3      0      0      0   100%
djpaddle/api.py                                                      21      0      6      0   100%
djpaddle/apps.py                                                      3      0      0      0   100%
djpaddle/context_processors.py                                        3      0      0      0   100%
djpaddle/fields.py                                                    6      0      0      0   100%
djpaddle/management/__init__.py                                       0      0      0      0   100%
djpaddle/management/commands/__init__.py                              0      0      0      0   100%
djpaddle/management/commands/djpaddle_sync_plans_from_paddle.py       8      0      2      0   100%
djpaddle/models.py                                                  107      0     18      4    97%
djpaddle/settings.py                                                 28      0      7      0   100%
djpaddle/signals.py                                                  18      0      0      0   100%
djpaddle/urls.py                                                      4      0      0      0   100%
djpaddle/utils.py                                                    27      0      4      0   100%
djpaddle/views.py                                                    20      0      8      0   100%
---------------------------------------------------------------------------------------------------
TOTAL                                                               248      0     45      4    99%
Coverage HTML written to dir cover

Required test coverage of 97% reached. Total coverage: 98.63%
================================================== 23 passed, 19 warnings in 1.12s ==================================================
lint installed: appdirs==1.4.4,attrs==19.3.0,black==19.10b0,click==7.1.2,flake8==3.8.1,importlib-metadata==1.6.0,isort==4.3.21,mccabe==0.6.1,pathspec==0.8.0,pycodestyle==2.6.0,pyflakes==2.2.0,regex==2020.5.14,toml==0.10.1,typed-ast==1.4.1,zipp==3.1.0
lint run-test-pre: PYTHONHASHSEED='3691936497'
lint run-test: commands[0] | flake8 /Users/mattpye/dev/dj-paddle
lint run-test: commands[1] | isort --check-only
Skipped 26 files
lint run-test: commands[2] | black /Users/mattpye/dev/dj-paddle --exclude=djpaddle/migrations/ --check
All done! ✨ 🍰 ✨
31 files would be left unchanged.
______________________________________________________________ summary ______________________________________________________________
  py37-django21-sqlite: commands succeeded
  py37-django22-sqlite: commands succeeded
  py37-django30-sqlite: commands succeeded
  py37-djangomaster-sqlite: commands succeeded
  py37-django22-checkmigrations: commands succeeded
  lint: commands succeeded
  congratulations :)
```

### CODE REVIEW

File: djpaddle/utils.py [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
-import phpserialize
 
+import phpserialize
 from Crypto.PublicKey import RSA
-from django.core.exceptions import ImproperlyConfigured
 
 try:
     from Crypto.Hash import SHA1
-except ImportError:
+except ImportError:  # pragma: no cover
```
Comment (pyepye) [inline djpaddle/utils.py:9]: I'm not a fan of `pragma: no cover` generally but I wasn't sure of another way to test this?

File: djpaddle/models.py [hunk 2]
```diff
@@ -191,8 +194,7 @@ def from_subscription_created(cls, payload):
     @classmethod
     def update_by_payload(cls, payload):
         data = cls._sanitize_webhook_payload(payload)
-        pk = data.pop("id")
-        return cls.objects.update_or_create(pk, defaults=data)
+        return cls.objects.update_or_create(id=data["id"], defaults=data)
```
Comment (pyepye) [inline djpaddle/models.py:197]: update_or_create

File: djpaddle/views.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ def post(self, request, *args, **kwargs):
 
         if alert_name in self.SUPPORTED_WEBHOOKS:
             signal = getattr(signals, alert_name, None)
-            if signal is not None:
+            if signal is not None:  # pragma: no cover
```
Comment (pyepye) [inline djpaddle/views.py:54]: I added `pragma: no cover` here as I wasn't sure of a way to get here via a test without making a change to either `signals.py` or `SUPPORTED_WEBHOOKS` as they currently have matching names (which they should).

File: djpaddle/models.py (hunk 2 above)
Comment (FPurchess) [reply djpaddle/models.py:197]: data.pop is used here to make sure the ID is not part of the defaults

**Comment by pyepye** [comment]
@FPurchess 👍 
I've put back in pop before the `update_or_create`

**Pull request merged**